thirdparty/CMakeLists.txt

```diff
@@ -59,10 +59,12 @@ if (CMAKE_EXECUTABLE_FORMAT STREQUAL "ELF")
     include_with_scope (libbacktrace.cmake)
 endif ()
 
-# We use non-standard C++ flags, so we can't just use GTest's CMakeLists.txt
-if (NOT GTEST_FOUND)
-    message (STATUS "Using bundled GTest")
-    include_with_scope (gtest.cmake)
+if (BUILD_TESTING)
+  # We use non-standard C++ flags, so we can't just use GTest's CMakeLists.txt
```

Comment:
Indentation is off: please use 4 spaces.

Reply:
Perhaps I wasn't clear: I meant 4 spaces _ per indentation level_, as per surrounding code.

You might also want to install an editorconfig plugin -- https://github.com/apitrace/apitrace/blob/master/.editorconfig#L12 -- for whatever editor you use, so these things are done automatically.

Reply:
I missed editorconfig, just installed with vim thanks !